src/main/java/games/strategy/engine/pbem/TripleAForumPoster.java

```diff
@@ -0,0 +1,202 @@
+package games.strategy.engine.pbem;
```

Comment:
Can we reasonably unit test this at all? What are your thoughts on how we could do that if at all?